### ISSUE TITLE
Unable to filter winlogbeats events by keywords field

### ISSUE THREAD

**Issue by vbohata** [elastic/beats#2237]
Version: v5.0.0-alpha5
Operating System: Windows Server 2012R2

I am unable to exclude events by keywords field. Config option " keywords: "Audit Success" " does not work, in beats log is: 

```
2016-08-11T10:05:43+02:00 WARN unexpected type []string in contains condition as it accepts only strings. 
2016-08-11T10:05:43+02:00 WARN unexpected type []string in contains condition as it accepts only strings. 
2016-08-11T10:05:43+02:00 WARN unexpected type []string in contains condition as it accepts only strings. 
2016-08-11T10:05:43+02:00 WARN unexpected type []string in contains condition as it accepts only strings. 
```

Config option " keywords: ["Audit Success"] " also does not work but there is no error in beats log.

Processors config:

```
processors:
  - drop_event:
      when:
        and:
          - equals:
              log_name: Security
          - contains:
              keywords: ["Audit Success"]
```

**Comment by spacewander** [comment]
Try `keywords: "Audit Success"`?

**Comment by vbohata** [comment]
I tried it, with this, there is error in log: unexpected type []string

**Comment by ruflin** [comment]
@vbohata Could you provide the full error message?

@andrewkroh Not sure if this could be related to https://github.com/elastic/beats/pull/2209 ?

**Comment by vbohata** [comment]
There is no more info in log. Just repeated "WARN unexpected type []string in contains condition as it accepts only strings." line.

**Comment by andrewkroh** [comment]
In the event log record, `keywords` is an array. The `contains` filter only accepts a string value as its input which is what causes the warning and the filter to not work. We should enhance it to check each element in the array.

And like @spacewander  said, it should be configured as `keywords: "Audit Success"`.

**Comment by andrewkroh** [comment]
I opened PR #2248 to make `contains` work on arrays of strings.

**Comment by andrewkroh** [comment]
This should be fixed in 5.0.
